Add support for various action keys to Linux keymap

Jules-Bertholet opened the pull request:
Adds support for Cut, Copy, Paste, Undo, Redo, New, Open, Save, and Find keys to the default keymap. These keys can be found on old keyboards, but also custom layouts like [Extend](https://dreymar.colemak.org/layers-extend.html).

Release Notes:

- Added support for the Cut, Copy, Paste, Undo, Redo, New, Open, Save, and Find keys to the default keymap.

Jules-Bertholet commented:
@cla-bot check

mgsloan commented:
This looks good, thank you!

I wondered which bindings would be preferred when displayed in the UI, and realized that within keymap files it's base on the lexicographic sort order.  I think it makes since to use the order within the JSON, preferring earlier entries, and so this change looks good for that.

I have a fix in progress for this, and would like to merge it before merging this PR.

notpeter commented on assets/keymaps/default-linux.json:
```diff
@@ -50,6 +52,7 @@
     "context": "Editor",
     "bindings": {
       "escape": "editor::Cancel",
+      "cancel": "editor::Cancel",
```
Do you have a physical `cancel` key on your keyboard?
I've never seen this and it feels pretty niche.

Jules-Bertholet replied:
I do not. The ones I currently have (with a custom layout, not physically on the keyboard) are Cut, Copy, Paste, Undo, and Redo. But I tried to be exhaustive with this PR.

notpeter replied:
Similar question about `execute` and `clear` keys.

notpeter replied:
I'd say let's only create default mapping for cut/copy/paste/new/open/save as these are not uncommon on Multimedia keyboards.

Let's hold off and not map `cancel`, `execute` and `clear`; they feel a little niche (and we can't test them).  Anyone who wants these should be able to easily map them themselves.

Jules-Bertholet replied:
I’ve removed Execute, Cancel, and Clear.

Jules-Bertholet committed:
Add support for various action keys to Linux keymap

Adds support for Cut, Copy, Paste, Undo, Redo, New, Open, Save, and Find keys
to the default keymap. These keys can be found on
old keyboards, but also custom layouts like Extend
(https://dreymar.colemak.org/layers-extend.html).

mgsloan commented:
Going to go ahead and merge this, will reorder them along with the rest of the bindings to match preferring the later binding for display
